### PR TITLE
[#2348] Improvement(docs): Add auth configuration doc for gvfs

### PR DESCRIPTION
**Pull request by xloya**
### What changes were proposed in this pull request?

Gvfs supports simple and oauth2 auth type, we need add a doc for how to configure them.

### Why are the changes needed?

Fix: #2836

### CODE REVIEW

File: docs/how-to-use-gvfs.md [hunk 1]
```diff
@@ -223,3 +228,49 @@ In order for tensorflow to support GVFS, you need to recompile the [tensorflow-i
...
+
+## Authentication
+
+Currently, Gravitino Virtual File System supports two kinds of authentication types to access Gravitino server: `simple` and `oauth2`.  
+`simple` type is the default authentication type in Gravitino Virtual File System.
+
+### How to use
+#### Use `simple` authentication type
+
+Please make sure that your Gravitino server also configure the `simple` authentication mode firstly.
```
Comment (qqqttt123) [inline docs/how-to-use-gvfs.md:240]: How to configure the user name in the simple mode?

Comment (xloya) [reply docs/how-to-use-gvfs.md:240]: Add it in the code example.

**Review by qqqttt123 — APPROVED** [review]
LGTM.

File: docs/how-to-use-gvfs.md [hunk 2]
```diff
@@ -223,3 +228,54 @@ In order for tensorflow to support GVFS, you need to recompile the [tensorflow-i
...
+
+Currently, Gravitino Virtual File System supports two kinds of authentication types to access Gravitino server: `simple` and `oauth2`.  
+`simple` type is the default authentication type in Gravitino Virtual File System.
+
+### How to use
+#### Use `simple` authentication type
+
+Please make sure that your Gravitino server also configure the `simple` authentication mode firstly.  
+Then, you can configure the Hadoop configuration like this:
+```java
```
Comment (jerryshao) [inline docs/how-to-use-gvfs.md:242]: Can you please add a blank line above for different paragraphs/blocks?

File: docs/how-to-use-gvfs.md [hunk 3]
```diff
@@ -223,3 +228,54 @@ In order for tensorflow to support GVFS, you need to recompile the [tensorflow-i
...
+FileSystem fs = filesetPath.getFileSystem(conf);
+```
+
+#### Use `oauth2` authentication type
+
+If you want to use `oauth2` authentication type for the Gravitino client in Gravitino Virtual File System,
+please refer to this document to complete the configuration of the Gravitino server and the OAuth server: [Security](./security.md).  
+
+Then, you can configure the Hadoop configuration like this:
+```java
```
Comment (jerryshao) [inline docs/how-to-use-gvfs.md:266]: Also here.

File: docs/how-to-use-gvfs.md [hunk 4]
```diff
@@ -223,3 +228,54 @@ In order for tensorflow to support GVFS, you need to recompile the [tensorflow-i
    ## list directory
    print(tf.io.gfile.listdir('gvfs://fileset/test_catalog/test_schema/test_fileset_1/'))
    ```
+
+## Authentication
+
+Currently, Gravitino Virtual File System supports two kinds of authentication types to access Gravitino server: `simple` and `oauth2`.
```
Comment (jerryshao) [inline docs/how-to-use-gvfs.md:234]: Also please make sure to remove all the trailing whitespace like here.

File: docs/how-to-use-gvfs.md [hunk 5]
```diff
@@ -223,3 +228,54 @@ In order for tensorflow to support GVFS, you need to recompile the [tensorflow-i
...
+// Configure the auth type to oatuh2.
+conf.set("fs.gravitino.client.authType", "oauth2");
+// Configure the oauth conifiguration.
+conf.set("fs.gravitino.client.oauth2.serverUri", "${your_oauth_server_uri}");
+conf.set("fs.gravitino.client.oauth2.credential", "${your_client_credential}");
+conf.set("fs.gravitino.client.oauth2.path", "${your_oauth_server_path}");
+conf.set("fs.gravitino.client.oauth2.scope", "${your_client_scope}");
+Path filesetPath = new Path("gvfs://fileset/test_catalog/test_schema/test_fileset_1");
+FileSystem fs = filesetPath.getFileSystem(conf);
+```
```
Comment (jerryshao) [inline docs/how-to-use-gvfs.md:281]: Can you please add an empty line in the bottom.

**Review by jerryshao — APPROVED** [review]
LGTM. @xloya thanks for your work.

**Pull request merged**